Lesson11-CSS_Positioning(class)/assets/index.css

```diff
@@ -0,0 +1,124 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #e9e1e0;
+  text-decoration: none;
+}
+h1 {
+  margin: 0;
+  padding: 0;
+}
+.container {
+  width: 500px;
+  height: 500px;
+  background: rgba(252, 208, 207, 0.5);
+  border: 0.974656px solid #fcd0cf;
+  box-sizing: border-box;
+  border-radius: 24px;
+  margin: auto;
+  margin-top: 10%;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+/* Avatar Left */
+/* .container-left {
+  border: solid 2px red;
+} */
+.container_left-avatar {
+  /* border: solid 2px blue; */
+  width: 70px;
+  height: 70px;
+  position: relative;
+}
+.avatar1-border {
```

Comment:
We need to have  '.main-avatar' class in this case. It will be BEM block name.
And inside this BEM block, we will have BEM elements.
`.main-avatar` BEM block
`.main-avatar__border` BEM element
`.main-avatar__background` BEM element

Reply:
Hi Alex,

I have changed my BEM - will you please check if it is right? - for lesson11 - Figma (Fluffy) Project as well as lesson11 - (class) The avatar we did in class last week Saturday.

Thank you very much.
Hope you solved the problem on your project!
Regards

Reply:
Good job. I'm usually using underscore for elements. But your naming is also possible.